### DISCUSSION TITLE
<Whole course>: <Drop language around "Knowledge check">

### DISCUSSION THREAD

**Discussion by bycdiaz** [TheOdinProject/curriculum#26669] [General]
### Describe your suggestion

Many learners are coming into the chat indicating they are reluctant to move on because they can't pass the knowledge checks. I think we should adjust this language in favor of something different.

An alternative is to just drop them completely. We already make mention of learning objectives at the start of lessons.

Would love a thought partner to join me in tossing this idea around! 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### (Optional) Discord Name

bycdiaz

**Comment by CouchofTomato** [comment]
@bycdiaz This would be better as a discussion. Are you happy either to convert or for me to?

**Reply by bycdiaz** [reply]
Thanks! I tried to do it the other day on the go and couldn't find the button on mobile. lol
